tables/xh-g1.utb

```diff
@@ -0,0 +1,61 @@
+# liblouis: isiXhosa/isiZulu braille code (grade 1)
+
+#-display-name: Xhosa uncontracted braille
+#-index-name: Xhosa, uncontracted
+
+#+locale: xh
+#+type: literary
+#+contraction: no
+#+grade: 1
+#+direction: both
+#+system: ubc-nguni
+
+# "UBC" stands for "Unified Braille Code" and is a generalization of UEB.
+# See https://sabrailleauthority.org.za/unified-braille-code-ubc.
+
+# The same table is used for isiXhosa and isiZulu. IsiXhosa and
+# isiZulu are two of the most widely spoken languages in South
+# Africa. They belong to the Nguni language family and are mutually
+# intelligible. IsiXhosa is predominent in the Eastern and Western
+# Cape provinces, but is spoken across South Africa. IsiZulu is
+# predominent in the KwaZulu-Natal province, but is also widely spoken
+# in South Africa.
+
+# The Nguni braille code is based on UEB, therefore the grade 1 system
+# is identical to UEB.
+
+# No additional codes such as Nemeth or computer braille codes are
+# used; the unified Nguni code is used also for technical material.
+
+# The table has been created by Christo de Klerk <cjdk@mweb.co.za>
+# according to the official code for these languages under the
+# auspices of and approved by the South African Braille Authority
+# (SABA).
+
+# The table is maintained by Christo de Klerk <cjdk@mweb.co.za>.
+
+#-author-name: Christo de Klerk
+#-maintainer-name: Christo de Klerk
+#-maintainer-email: cjdk@mweb.co.za
+#-copyright: 2021, South African Braille Authority
+#-license: LGPLv2.1
+
+# Copyright (C) 2021 South African Braille Authority <http://www.sabrailleauthority.org.za>
```

Comment:
I'd prefer if the copyright and license info was at the top. The `licensecheck` tool only checks "the start of the file for text belonging to various licenses"

Reply:
What is the licensecheck tool and why do we need to conform with it?

Reply:
> What is the licensecheck tool

The licensecheck tool comes with Debian (the original site for it is apparently on [CPAN](https://metacpan.org/pod/App::Licensecheck). It is a perl script that checks your source code for known licenses. I use(d) it in the wiki page [Licensing of liblouis tables](https://github.com/liblouis/liblouis/wiki/Licensing-of-liblouis-tables) to check which tables have what license.

>  and why do we need to conform with it?

we don't really but knowing what license a table has helps us to determine whether or not we are entitled to distribute it.

Reply:
What about all the other files? They don't all have the license at the top?

Should we have a Github flow that does this check?

Reply:
@egli

Reply:
Yes, why not. `licensecheck` installable from the ubuntu image

Reply:
But what is the check testing? If I run it on master I get some tables that are not licensed under  LGPL-2.1+:
``` console
$ cd ~/src/liblouis/tables
$ licensecheck --copyright --check='\.(ctb|utb|cti|uti|dis)' --deb-fmt --encoding utf8 --machine * | grep -v LGPL-2.1+
Es-Es-G0.utb	LGPL-3+	2012 José Enrique Fernández del Campo and Juan Carlos Buño Suárez <quetzatl@eresmas.net>
es-g2.ctb	LGPL	2018 Juan Pablo Bello / 1999-2006 The BRLTTY Team
et-g0.utb	LGPL-3+	2012 Mesar Hameed <mesar.hameed@gmail.com> / 2012 Jürgen Dengo <jyrgen.dengo@gmail.com> / 1999-2006 The BRLTTY Team
ethio-g1.ctb	UNKNOWN	2012, 2018 Dr. Tamru E. Belay <tamru@sympatico.ca> / 2012, 2015 Mesar Hameed <mesar.hameed@gmail.com>
2008:2008 is pointless at /usr/share/perl5/String/Copyright.pm line 194.
is-chardefs6.cti	LGPL-3+	2010 INIB (Iceland National Institute for the Blind)
is-chardefs8.cti	LGPL-3+	2009 INIB (Iceland National Institute for the Blind)
it-it-comp8.utb	LGPL-3+	2012, 2016 Simone Dal Maso <simone.dalmaso@juvox.it> / 2011 Caterina Avoledo <catery81@yahoo.it>
pl-pl-comp8.ctb	LGPL-3+	2019-2021 Łukasz Golonka <lukasz.golonka@mailbox.org> / 2015 Harpo Sp. z o. o. <info@harpo.com.pl>
pt-pt-g2.ctb	LGPL-3+	2009 Carlos Ferreira <cferreira9886@gmail.com>
ru-litbrl-detailed.utb	UNKNOWN	2020 Bert Frees <bertfrees@gmail.com> / 2020 Andrey Yakuboy <andrewia2002@yandex.ru>
2008:2008 is pointless at /usr/share/perl5/String/Copyright.pm line 194.
sr-chardefs.cti	LGPL-3+	2011 Peter Engström <peter.engstrom@indexbraille.com> / 2011 Monk Jeremiah, Manastir Decani <visokidecani@gmail.com>
sr-g1.ctb	LGPL-3+	2011 Peter Engström <peter.engstrom@indexbraille.com> / 2011 Monk Jeremiah, Manastir Decani <visokidecani@gmail.com>
zh-hk.ctb	UNKNOWN	2009 KM Yuen, Keny Yuen, Alex Ho and Eric Yip / 2009 JJB Software, Inc. www.jjb-software.com
```

Reply:
It would check that all files are in the format that you would like them to be, whatever that is, and that they have a license and copyright. I'm not sure what the format should be. You said the license needs to be "at the start of the file" but what does that mean? The tool apparently detects LGPLv3 in Es-Es-G0.utb even though the license it not all the way at the top.

Reply:
Hehe, the option `--lines INT` lets you specify the number of lines to search. Default is 60. `--lines 0` means search the whole file. But the default is probably OK. It should not be to far down as far as I'm concerned.